packages/components/src/toggle-group-control/test/index.tsx

```diff
@@ -80,6 +80,13 @@ const optionsWithTooltip = (
 		/>
 	</>
 );
+const optionsWithDisabledOption = (
+	<>
+		<ToggleGroupControlOption value="pizza" label="Pizza" />
+		<ToggleGroupControlOption value="rice" label="Rice" disabled />
+		<ToggleGroupControlOption value="pasta" label="Pasta" />
```

Comment:
@ramonjd just making sure you're ok with those food options

Reply:
Sorry, I was away last week.

I would have preferred risotto instead of just "rice", but I appreciate you asking. 😆

Reply:
Ah, good point. Feel free to open a PR